src/backend/src/services/users.services.ts

```diff
@@ -576,13 +578,34 @@ export default class UsersService {
    * @returns the unread notifications of the user
    */
   static async getUserUnreadNotifications(userId: string, organization: Organization) {
+    const unreadNotifications = await prisma.notification.findMany({
+      where: {
+        users: {
```

Comment:
i would assume youd also only want notifcations for this organization

Reply:
and non deleted notifications? or are we hard deleting those

Reply:
@caiodasilva2005 can you take a look at this?

Reply:
This was addressed in slack so I believe this is all good